site/docs/v1/tech/tutorials/integrate-expressjs.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: doc
+title: Integrate Your Express.js Application With FusionAuth
+description: Integrate your Express.js application with FusionAuth
+navcategory: getting-started
+prerequisites: Node.JS
+language: JavaScript
+technology: Express.js
+---
+
+{% include docs/integration/_intro.md %}
+
+## Prerequisites
+
+{% include docs/integration/_prerequisites.md %}
+
+## Download and Install FusionAuth
+
+{% include docs/integration/_install-fusionauth.md %}
+
+## Create a User and an API Key
+
+{% include docs/integration/_add-user.md language=page.language %}
+
+## Configure FusionAuth
+
+Next, you need to set up FusionAuth. This can be done in different ways, but we’re going to use the {{page.language}} client library. The below instructions use `npm` on the command line, but you can use the client library with an IDE of your preference as well.
+
+First, make a directory:
+
+```shell
+mkdir setup-fusionauth && cd setup-fusionauth
+```
+
+Now, copy and paste the following file into `package.json`.
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/package.json %}
+```
+
+Now you need to install the dependencies in `package.json`.
+
+```shell
+npm install
+```
+
+Then copy and paste the following file into `setup.js`. This file uses the [FusionAuth API](/docs/v1/tech/apis/) to configure an Application and more to allow for easy integration. 
+
+```javascript
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/typescript/setup.js %}
```

Comment:
This isn't going to work because it has a 5173 port in the redirect urls. Please create a new script called setup-express.js in that same repo/directory. You can also remove the CORS stuff, because that isn't needed for a server side integration.

Reply:
Done in FusionAuth/fusionauth-example-client-libraries#4